ROLES.md

```diff
@@ -151,65 +138,24 @@ All members are encouraged to help with the code review burden, although each PR
-
-## Reviewer
-
-Reviewers are able to review code for quality and correctness on some part of the project. They are knowledgeable about both the codebase and software engineering principles. Reviewer status is scoped to a part of the codebase.
-
-All members are encouraged to help with the code review burden, although each PR must be reviewed by an official reviewer.
-
-Note that acceptance of code contributions requires at least one approver in addition to the assigned reviewers.
-
-### Requirements
-
-The following apply to the part of the codebase for which one would be a reviewer in an OWNERS file:
-
-* Member for at least 3 months
-
-* Primary reviewer for at least 5 PRs to the codebase
-
-* Reviewed or merged at least 20 substantial PRs to the codebase
-
-* Knowledgeable about the codebase
-
-* Sponsored by an area approver
-
-    * With no objections from other approvers
-
-    * Done through PR to update the OWNERS file
-
-* May either self-nominate, be nominated by an approver in this area, or be nominated by a robot
-
-### Responsibilities and privileges
-
-The following apply to the part of the codebase for which one would be a reviewer in an OWNERS file:
-
-* Responsible for project quality control via [code reviews](https://github.com/istio/community/blob/master/REVIEWING.md)
-
-    * Focus on code quality and correctness, including testing and factoring
-
-    * May also review for more holistic issues, but not a requirement
-
-* Expected to be responsive to review requests
-
-* Assigned PRs to review related to area of expertise
-
-* Assigned test bugs related to area of expertise
+Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
 ## Approver
 
-Code approvers are able to both review and approve code contributions. While code review is focused on code quality and correctness, approval is focused on holistic acceptance of a contribution including: backwards / forwards compatibility, adhering to API and flag conventions, subtle performance and correctness issues, interactions with other parts of the system, etc. Approver status is scoped to a part of the codebase.
+Code approvers are able to both review and approve code contributions. While code review is focused on code quality and correctness,
+approval is focused on holistic acceptance of a contribution including: backwards / forwards compatibility, adhering to API and flag
+conventions, subtle performance and correctness issues, interactions with other parts of the system, etc. Approver status is scoped to a
+part of the codebase.
 
 ### Requirements
 
 The following apply to the part of the codebase for which one would be an approver in an OWNERS file:
 
-* Reviewer of the codebase for at least 3 months
+* Member for at least 3 months
 
-* Primary reviewer for at least 10 substantial PRs to the codebase
+* Merged at least 30 substantial PRs to the codebase
```

Comment:
`merged` is ambiguous, I think you mean authored but could be (mis)interpreted as clicked the squash&merge button. Also depending on your definition of "substantial", has anyone done 30 substantial PRs ? (aka 30 features)

Reply:
Switched to "Contributed"